lisa/tests/lisa/test_conf.py

```diff
@@ -0,0 +1,169 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2018, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+import copy
+from unittest import TestCase
+
+from lisa.utils import MultiSrcConf, KeyDesc, LevelKeyDesc, TopLevelKeyDesc, IntList
+from lisa.tests.lisa.utils import StorageTestCase, HOST_PLAT_INFO, HOST_TARGET_CONF
+
+""" A test suite for the MultiSrcConf subclasses."""
+
+
+class TestMultiSrcConfBase:
+    """
+    A test class that exercise various APIs of MultiSrcConf
+    """
+    def test_serialization(self):
```

Comment:
So at the moment all of our test methods have a one-liner docstring, and this is what nosetest will print when there are failures.

e.g.:

https://github.com/ARM-software/lisa/blob/next/lisa/tests/lisa/test_test_bundle.py#L61
gives
ERROR: Test that creating a bundle from a target works

Removing the docstring gives me:
ERROR: test_target_init (lisa.tests.lisa.test_test_bundle.BundleCheck)

I was about to suggest adding a docstring to your test methods, but now I'm wondering if having no docstring just doesn't make the nosetest ouput clearer...